domain/schema/model/sql/0011-storage.sql

```diff
@@ -169,10 +172,10 @@ INSERT INTO storage_provisioning_status VALUES
 
 CREATE TABLE storage_volume (
     uuid TEXT NOT NULL PRIMARY KEY,
+    volume_id TEXT NOT NULL,
     life_id INT NOT NULL,
-    name TEXT NOT NULL,
+    name TEXT,
```

Comment:
Remind me; what's different between name and volume ID? When would name be NULL?

Reply:
Ah, I was conflating name and world wide name which we store in the wwn column. We don't need the former, it can be dropped.
Both filesystem and volume have an ID, which is a monotonically increasing number generated when they are created. These form part of the respective tag values, eg "2/4" is filesystem id 4 on machine 2.